be1-go/channel/lao/mod.go

```diff
@@ -703,16 +703,12 @@ func (c *Channel) createAndSendLAOGreet() error {
 		return xerrors.Errorf("failed to marshal the organizer key: %v", err)
 	}
 
-	peers := []messagedata.Peer{}
+	peersInfo := c.hub.GetPeersInfo()
```

Comment:
Just taking a step back, why wouldn't `GetPeersInfo()` already return a unique list of peers ? 
Do we ever need it to return duplicates ? I might very well be missing something here and there could be a perfectly good reason, so think of this as a genuine question and not an invitation to change "because @pierluca said so" 😅

Reply:
You're right, I moved the checks to the `GetAllPeersInfo()` in `Peers` to make it return a unique list of server info.